src/encoded/types/dataset.py

```diff
@@ -328,6 +328,27 @@ class Annotation(FileSet, CalculatedVisualize):
         },
     }
 
+    @calculated_property(schema={
+        "title": "Files",
+        "type": "array",
+        "items": {
+            "type": "string",
+            "linkTo": "File",
+        },
+    })
+    def files(self, request, original_files, status):
+        if status in ('released', 'archived'):
```

Comment:
might be better to put these sets in vars to avoid hardcoding in two locations.
```
good_status = ('released', 'archived')
bad_status = ('revoked', 'deleted', 'replaced')
if status in good_status:
    etc...
else:
    etc...

Reply:
I could refactor this, it would probably have to be refactored in types/experiment.py and even elsewhere.

Reply:
@caseylitton This was update as per your review, and just this method.